Gems/ROS2/Code/Source/Clock/SimulationClock.cpp

```diff
@@ -0,0 +1,45 @@
+#include "SimulationClock.h"
```

Comment:
Maybe it should also contain the copyright note?

Reply:
Missed this one, thank you!